tests/performance/realtime.go

```diff
@@ -43,24 +40,27 @@ func byStartingTheVMI(vmi *v1.VirtualMachineInstance, virtClient kubecli.Kubevir
 	tests.WaitForSuccessfulVMIStart(vmi)
 }
 
-func byConfiguringTheVMIForRealtime(vmi *v1.VirtualMachineInstance, realtimeMask string) {
-	vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{
-		k8sv1.ResourceMemory: memoryRequest,
-		k8sv1.ResourceCPU:    resource.MustParse("2"),
-	}
-	vmi.Spec.Domain.Resources.Requests = k8sv1.ResourceList{
-		k8sv1.ResourceMemory: memoryRequest,
-		k8sv1.ResourceCPU:    resource.MustParse("2"),
-	}
-	vmi.Spec.Domain.CPU = &v1.CPU{
-		Model:                 "host-passthrough",
-		DedicatedCPUPlacement: true,
-		Realtime:              &v1.Realtime{Mask: realtimeMask},
-		NUMA:                  &v1.NUMA{GuestMappingPassthrough: &v1.NUMAGuestMappingPassthrough{}},
-	}
-	vmi.Spec.Domain.Memory = &v1.Memory{
-		Hugepages: &v1.Hugepages{PageSize: "2Mi"},
-		Guest:     &memoryRequest,
+func withRelatimeConfiguration(realtimeMask string) libvmi.Option {
```

Comment:
I think `realtimeMask` is not used (even before), can you add a commit before this one that removes it?

Reply:
I will to it in future PR